dll/cpl/usrmgr/users.c

```diff
@@ -666,16 +706,52 @@ UsersPageProc(HWND hwndDlg,
     switch (uMsg)
     {
         case WM_INITDIALOG:
+        {
+            LONG lResultInit;
+            HKEY hKeyInit;
+            WCHAR szAutoAdminLogonValueInit[2];
+            DWORD dwTypeInit;
+            DWORD dwSizeInit = sizeof(szAutoAdminLogonValueInit);
+            BOOL bRequireLogonInit = TRUE;
+
             pUserData = HeapAlloc(GetProcessHeap(), HEAP_ZERO_MEMORY, sizeof(USER_DATA));
             SetWindowLongPtr(hwndDlg, DWLP_USER, (LONG_PTR)pUserData);
 
             pUserData->hPopupMenu = LoadMenu(hApplet, MAKEINTRESOURCE(IDM_POPUP_USER));
 
             OnInitDialog(hwndDlg);
+			
+            lResultInit = RegOpenKeyExW(HKEY_LOCAL_MACHINE,
+                                        L"SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion\\Winlogon",
+                                        0, KEY_READ, &hKeyInit);
+            if (lResultInit != ERROR_SUCCESS)
+            {
+                CheckDlgButton(hwndDlg, IDC_USERS_STARTUP_REQUIRE, BST_CHECKED);
+                break; 
+            }
+
+            lResultInit = RegQueryValueExW(hKeyInit,
+                                           L"AutoAdminLogon",
+                                           NULL,
+                                           &dwTypeInit,
+                                           (LPBYTE)szAutoAdminLogonValueInit,
+                                           &dwSizeInit);
+            RegCloseKey(hKeyInit);
+
+            if (   lResultInit == ERROR_SUCCESS
+                && dwTypeInit == REG_SZ
+                && wcscmp(szAutoAdminLogonValueInit, L"1") == 0)
```

Comment:
```suggestion
            if (lResultInit == ERROR_SUCCESS && dwTypeInit == REG_SZ &&
                wcscmp(szAutoAdminLogonValueInit, L"1") == 0)
```

Reply:
Please, re-apply this change.

> I would rather leave it as it now for better readability.

That is not ReactOS style: see examples on https://reactos.org/wiki/Coding_Style#Braces